src/Icons/doc/index.rst

```diff
@@ -43,6 +77,61 @@ define the HTML attributes added to the ``<svg>`` element:
     {{ ux_icon('user-profile', {height: '16px', width: '16px', aria-hidden: true}) }}
     {# renders <svg height="16" width="16" aria-hidden="true"> ... </svg> #}
 
+.. code-block:: yaml
+
+    # config/packages/ux_icons.yaml
+    ux_icons:
+        iconify:
+            enabled: true
+            on_demand: true
+            endpoint: 'https://api.iconify.design'
+
+Icon Sets
+~~~~~~~~~
+
+Icon Sets many icon sets available, each with their own unique style and set of icons
+provide a wide range of icons for different purposes, while maintaining a consistent
+look and feel across your application.
+
+With more than 200,000 icons available, you can find the perfect icon for your project.
+
+Here are some of the most popular icon sets available:
+
+===================== ======= ========= ======= =======================
+Icon Set               Icons  License   Prefix       Example
+===================== ======= ========= ======= =======================
+`Octicons`_              700   MIT       oct        oct:checked
+`Bootstrap Icons`_       2000  MIT       bi         bi:check
+`Boxicons`_              1700  MIT       bx         bx:check
+`Flowbite`_              1000  MIT       flowbite   flowbite:check
+`FontAwesome Icons`_     7000  CC BY 4.0 fa        fa:check
+`Heroicons`_             300   MIT       hero       hero:check
+`Iconoir`_               1500  MIT       iconoir    iconoir:check
+`Ionicons`_              1300  MIT       ion        ion:check
+`Lucide Icons`_          1500  MIT       lucide     lucide:check
+`Material Design Icons`_ 5000  Apache 2  mdi        mdi:check
+`Octicons`_              200   MIT       oct        :oct:check
+`Phosphor Icons`_        7000  MIT       phosphor   phosphor:check
+`Tabler Icons`_          5200  MIT       tabler     tabler:check
+===================== ======= ========= ======= =======================
+
+- `Bootstrap Icons <https://icons.getbootstrap.com/>`_
+- `Boxicons <https://boxicons.com/>`_
+- `Flowbite <https://github.com/themesberg/flowbite>`_
+- `FontAwesome Icons <https://github.com/FortAwesome/Font-Awesome>`_
+- `Heroicons <https://github.com/tailwindlabs/heroicons>`_
+- `Iconoir <https://github.com/iconoir-icons/iconoir>`_
+- `Ionicons <https://github.com/ionic-team/ionicons>`_
+- `Lucide Icons <https://github.com/lucide-icons/lucide>`_
+- `Material Design Icons <https://github.com/google/material-design-icons>`_
+- `Octicons <https://github.com/primer/octicons/>`_
+- `Phosphor Icons <https://github.com/phosphor-icons/homepage>`_
+- `Tabler Icons <https://github.com/tabler/tabler-icons>`_
+
+
+To see the full list of available icon sets, visit `ux.symfony.com/icons`_.
```

Comment:
There are extra blank lines before and after this phrase. Sorry for all this nitpicking, but the parser is pretty strict.